metadata/collector/collector.go

```diff
@@ -424,7 +423,7 @@ func (c *Collector) getCtiSchema(base *raml.BaseShape, ctiSchema *raml.DomainExt
 }
 
 func (c *Collector) readCtiCti(base *raml.BaseShape) ([]string, error) {
```

Comment:
```suggestion
func (c *Collector) readMetadataCti(base *raml.BaseShape) ([]string, error) {
```

Reply:
Good point, thanks. How about using UPPER case for initialisms (`readMetadataCti` -> `readMetadataCTI`)? It's a [common practice in Go](https://go.dev/wiki/CodeReviewComments#initialisms).